lib/logger.js

```diff
@@ -1,68 +1,64 @@
-'use strict'
-
 const path = require('path')
 const { getCorrelationId } = require('@first-lego-league/ms-correlation')
 
 const LOG_LEVELS = require('./log-levels')
 
-function Logger () {
-  if (!(this instanceof Logger)) {
-    return new Logger()
+class Logger {
+  constructor () {
+    this.moduleName = path.basename(path.resolve())
+    this.LOG_LEVELS = LOG_LEVELS
+    this.logLevel = process.env.LOG_LEVEL || LOG_LEVELS.DEBUG
   }
 
-  // As a rule of thumb, launcher would run the process in
-  // the module directory which is in the name of the module
-  this.moduleName = path.basename(path.resolve())
-
-  this.LOG_LEVELS = LOG_LEVELS
-  this.setLogLevel(process.env.LOG_LEVEL || LOG_LEVELS.DEBUG)
-}
-
-Logger.prototype.setLogLevel = function (logLevel) {
```

Comment:
We should either add this method with deprecated properties or make sure next version is major version.